docker-app/qfieldcloud/subscription/admin.py

```diff
@@ -105,10 +105,7 @@ class SubscriptionAdmin(QFieldCloudModelAdmin):
         "status",
     )
 
-    list_filter = (
-        SubscriptionPeriodFilter,
-        "status",
-    )
+    list_filter = (SubscriptionPeriodFilter, "status", "plan")
```

Comment:
```suggestion
    list_filter = (
        SubscriptionPeriodFilter,
        "status",
        "plan",
     )
```

Reply:
Also if you have a trailing comma?

Reply:
Does not make sense a longer line to be kept in the same line, while the original tuple was in two lines. Also checked locally, black does respect the trailing commas.

Note we always prefer to use trailing commas on lists/tuples/dicts/whatever structs that have the potential to grow in time.